src/h5web/visualizations/heatmap/DomainSlider.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import ReactSlider from 'react-slider';
+import shallow from 'zustand/shallow';
+import { format } from 'd3-format';
+import styles from './DomainSlider.module.css';
+import { useHeatmapStore, Domain } from './store';
+
+const EXTEND_PERCENTAGE = 0.2;
+const MIN_DECIMALS = 1;
```

Comment:
The floating point number troubles are strong in this one...

Number values are rounded and formatted to `MIN_DECIMALS` to avoid these.